src/main/java/net/robinfriedli/aiode/concurrent/ThreadContext.java

```diff
@@ -220,6 +220,12 @@ public static void clear() {
             get().clear();
         }
 
+        /**
+         * Should be called to free the ThreadLocal variable since the thread may be reused later. Since the thread is
+         * kept in a thread pool, the garbage collector cannot determine that the variable should be freed.
+         */
+        public static void remove() { THREAD_CONTEXT.remove(); }
```

Comment:
Maybe this would method be more readable (and enterprise-y) if you called it `removeThreadLocal()` or `removeThreadContext()`.

Reply:
Since we're really freeing a resource, what do you think of changing it to `free` instead? I feel like `removeThreadLocal` and `removeThreadContext` are kind of redundant…

Reply:
Sure, that works.